version/version.go

```diff
@@ -14,6 +14,30 @@
 
 package version
```

Comment:
the root version dir seems like the wrong place for this file.  should it not be pkg/version/version.go?

Reply:
I struggled with this a bit but in practice it will probably be ok. The difference is that pkg/version is a general library for version handling, while version/version deals specifically with specifying the binary version of the operator and the supported version mappings. 
As such, the release team would be going into version, while developers would stick to pkg/version. 
We can discuss this and change it if you have some good idea here.